Packs/AtlassianJiraServiceManagement/Integrations/AtlassianJiraServiceManagement/AtlassianJiraServiceManagement.py

```diff
@@ -0,0 +1,831 @@
+        return self._http_request(method='GET', url_suffix=f'/object/{object_id}')
+
+    def search_objects(self, ql_query: str, include_attributes: bool, page: int, page_size: int, limit: int = None):
+        params = {
+            'qlQuery': ql_query,
+            'includeAttributes': include_attributes,
+            'page': page,
+            'resultsPerPage': limit if limit else page_size
+        }
+
+        return self._http_request(
+            method='GET',
+            url_suffix='/aql/objects',
+            params=params
+        )
+
+    def get_comment_list(self, object_id: str):
+        return self._http_request(method='GET', url_suffix=f'/comment/object/{object_id}')
+
+    def get_object_connected_tickets(self, object_id: str):
+        return self._http_request(method='GET', url_suffix=f'/objectconnectedtickets/{object_id}/tickets')
+
+    def get_object_attachment_list(self, object_id):
+        return self._http_request(method='GET', url_suffix=f'/attachments/object/{object_id}')
+
+    def get_file(self, file_url):
+        return self._http_request(
+            method='GET',
+            full_url=file_url,
+            resp_type='response'
+        )
+
+    def create_object(self, json_data):
+        return self._http_request(
+            method='POST',
+            url_suffix='/object/create',
+            json_data=json_data
+        )
+
+    def create_comment(self, object_id, comment):
+        url_suffix = '/comment/create'
+        json_data = {
+            'object_id': object_id,
+            'comment': comment
+        }
+        return self._http_request(
+            method='POST',
+            url_suffix=url_suffix,
+            json_data=json_data
+        )
+
+    def send_file(self, object_id, file_path):
+        return self._http_request(
+            method='POST',
+            url_suffix=f'/attachments/object/{object_id}',
+            files={'file': (file_path, open(file_path, 'rb'))}
+        )
+
+    def update_object(self, object_id: str, json_data: Dict[str, Any]):
+        return self._http_request(
+            method='PUT',
+            url_suffix=f'/object/{object_id}',
+            json_data=json_data
+        )
+
+    def delete_object(self, object_id: str):
+        return self._http_request(
+            method='DELETE',
+            url_suffix=f'/object/{object_id}'
+        )
+
+    def remove_file(self, attachment_id):
+        return self._http_request(
+            method='DELETE',
+            url_suffix=f'/attachments/{attachment_id}'
+        )
+
+
+''' HELPER FUNCTIONS '''
+
+
+def pascal_case(s: str) -> str:
+    """ Convert a string to PascalCase. """
+    words = s.split('_')
+    return ''.join(word[:1].upper() + word[1:] for word in words)
+
+
+def convert_keys_to_pascal(objects: List[dict[str, Any]], key_mapping: Optional[dict[str, str]] = None) -> List[dict[str, Any]]:
+    """
+    Convert keys of objects in a list to PascalCase, with optional key mapping.
```

Comment:
generally we shouldnt be changing the context outputs. we want to dump it in the context exactly like we got them from the api, same cases and conventions

Reply:
done